tools/parse/src/main/scala/org/allenai/nlpstack/parse/poly/eval/ParseAnalyzer.scala

```diff
@@ -0,0 +1,116 @@
+package org.allenai.nlpstack.parse.poly.eval
+
+import org.allenai.nlpstack.parse.poly.polyparser.PolytreeParse
+
+/** A ParseAnalyzer maps a candidate parse to an "analysis", i.e. a histogram.
+  *
+  * For instance, this histogram may be the count of mistaken arclabels or part-of-speech tags.
+  */
+trait ParseAnalyzer extends (PolytreeParse => Map[String, Double]) {
```

Comment:
We currently don't, but since this code doesn't need to be performant (and because I can foresee cases in which we'd want Doubles), I just made it Double.